packages/scoutgame/src/points/getClaimableTokensWithSources.ts

```diff
@@ -151,6 +152,16 @@ export async function getClaimableTokensWithSources(userId: string): Promise<Unc
     }))
     .filter((proof) => proof.amount > 0 && proof.proofs.length > 0 && !claimedWeeks.includes(proof.week));
 
+  const partnerRewardPayoutCount = await prisma.partnerRewardPayout.count({
```

Comment:
i just realized this method is doing a lot of db lookups in series, would be worth refactoring at some point, or at least trying to add new queries in parallel.

Reply:
for now I can just add the new partner rewards payout query in parallel